guide/src/componentDocs/CheckableAvatar/CheckableAvatarEx.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from "react";
+import { CheckableAvatar, Element } from "cyverse-ui";
+import { pad, marg } from "cyverse-ui/styles";
+import { Paper, Avatar } from "material-ui";
+
+class CheckableAvatarEx extends Component {
+    state = {
+        isHovered: false,
```

Comment:
This is not the right way to set state in a component. You are adding the attributes to the class, not to the instance of each clash, which you would do in a constructor.